client/src/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -379,3 +390,180 @@ export function withoutSearch() {
+            name: 'workflow-tasks',
+            label: 'Workflow tasks',
+            url: '/admin/reports/workflow_tasks/',
+            icon_name: 'thumbtack',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'site-history',
+            label: 'Site history',
+            url: '/admin/reports/site-history/',
+            icon_name: 'history',
+            classnames: '',
+          }),
+        ]
+      ),
+      new SubMenuItemDefinition(
+        {
+          name: 'settings',
+          label: 'إعدادات',
+          icon_name: 'cogs',
+          classnames: '',
+        },
+        [
+          new LinkMenuItemDefinition({
+            name: 'workflows',
+            label: 'Workflows',
+            url: '/admin/workflows/list/',
+            icon_name: 'tasks',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'workflow-tasks',
+            label: 'Workflow tasks',
+            url: '/admin/workflows/tasks/index/',
+            icon_name: 'thumbtack',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'users',
+            label: 'مستخدمين',
+            url: '/admin/users/',
+            icon_name: 'user',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'groups',
+            label: 'مجموعات',
+            url: '/admin/groups/',
+            icon_name: 'group',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'sites',
+            label: 'مواقع',
+            url: '/admin/sites/',
+            icon_name: 'site',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'collections',
+            label: 'مجموعات',
+            url: '/admin/collections/',
+            icon_name: 'folder-open-1',
+            classnames: '',
+          }),
+          new LinkMenuItemDefinition({
+            name: 'redirects',
+            label: 'اعادة التوجيهات',
+            url: '/admin/redirects/',
+            icon_name: 'redirect',
+            classnames: '',
+          }),
+        ]
+      ),
+    ],
+    [
+      new LinkMenuItemDefinition({
+        name: 'account',
+        label: 'حساب',
+        url: '/admin/account/',
+        icon_name: 'user',
+        classnames: '',
+      }),
+      new LinkMenuItemDefinition({
+        name: 'logout',
+        label: 'تسجيل الخروج',
+        url: '/admin/logout/',
+        icon_name: 'logout',
+        classnames: '',
+      }),
+    ],
+    {
+      name: 'Admin',
+      avatarUrl: 'https://gravatar.com/avatar/e31ec811942afbf7b9ce0ac5affe426f?s=200&d=robohash&r=x',
+    }
+  );
+}
+
+export function rightToLeft() {
+  return renderSidebarStory([
+    wagtailBrandingModule(),
+    searchModule(),
+    arabicMenuModule(),
+  ], { rtl: true, strings: STRINGS_AR });
```

Comment:
We don't set this `rtl` based on language at present, do we? Is this for a future PR?

Reply:
Yep, future PR